returnn/torch/engine.py

```diff
@@ -51,7 +51,15 @@ def __init__(self, config: Config):
         self._save_model_epoch_interval = 1
         self._updater = None  # type: Optional[Updater]
 
-        self._device = "cuda" if torch.cuda.is_available() and torch.cuda.device_count() > 0 else "cpu"
+        device = config.typed_value("device")
+        if (device == "gpu"):
+            print("Use of deprecated keyword 'gpu' to set device. Using 'cuda' instead.")
```

Comment:
```suggestion
            print("Use of deprecated keyword 'gpu' to set device. Using 'cuda' instead.", file=log.v1)
```

Returnn always uses specific log levels to print messages.